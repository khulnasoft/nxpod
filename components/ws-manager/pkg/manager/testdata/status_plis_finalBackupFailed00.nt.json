{
    "description": "Do not append the finalBackupFailure multiple times to the workspace failure message",
    "pod": null,
    "plis": {
        "metadata": {
            "name": "plis-4a385be8-2be6-4168-861d-ebfc4141627c",
            "namespace": "staging-gh-2510",
            "selfLink": "/api/v1/namespaces/staging-gh-2510/configmaps/plis-4a385be8-2be6-4168-861d-ebfc4141627c",
            "uid": "b6bf6c77-9408-11e9-97df-42010a8402a0",
            "resourceVersion": "110643424",
            "creationTimestamp": "2019-06-21T09:41:15Z",
            "labels": {
                "app": "nxpod",
                "component": "workspace",
                "gpwsman": "true",
                "headless": "false",
                "metaID": "fddc2a09-43a1-407e-abe1-d232bf4caaac",
                "owner": "4447d447-c11d-4a4a-9335-d47a952e3574",
                "workspaceID": "4a385be8-2be6-4168-861d-ebfc4141627c",
                "workspaceType": "regular"
            },
            "annotations": {
                "nxpod/id": "4a385be8-2be6-4168-861d-ebfc4141627c",
                "nxpod/servicePrefix": "fddc2a09-43a1-407e-abe1-d232bf4caaac",
                "nxpod/never-ready": "true",
                "nxpod/plis": "{\"finalBackupComplete\":true,\"finalBackupFailure\":\"cannot finalize workspace content: not implemented\",\"stoppingSince\":\"2019-06-21T09:41:33.690349983Z\",\"lastPodStatus\":{\"id\":\"4a385be8-2be6-4168-861d-ebfc4141627c\",\"metadata\":{\"owner\":\"4447d447-c11d-4a4a-9335-d47a952e3574\",\"metaId\":\"fddc2a09-43a1-407e-abe1-d232bf4caaac\"},\"spec\":{\"workspaceImage\":\"eu.gcr.io/nxpod-dev/workspace-images/b24b4698d67f493b801d17196870fd8a422ffa1e/eu.gcr.io/nxpod-dev/workspace-full:sha256-535009d8cf429001e17f0f6388f33065c53cb70a62904800aa3f424403c7cb7e\",\"url\":\"http://fddc2a09-43a1-407e-abe1-d232bf4caaac.ws-eu.gh-2510.staging.nxpod.khulnasoft.com\"},\"phase\":5,\"runtime\":{\"node_name\":\"foobar\"},\"conditions\":{\"failed\":\"last backup failed: cannot finalize workspace content: not implemented\",\"finalBackupComplete\":1}}}"
            }
        }
    }
}