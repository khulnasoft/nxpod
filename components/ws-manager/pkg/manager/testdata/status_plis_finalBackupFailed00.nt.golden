{
    "status": {
        "id": "4a385be8-2be6-4168-861d-ebfc4141627c",
        "metadata": {
            "owner": "4447d447-c11d-4a4a-9335-d47a952e3574"
        },
        "spec": {
            "url": "http://fddc2a09-43a1-407e-abe1-d232bf4caaac.ws-eu.gh-2510.staging.nxpod.io"
        },
        "phase": 6,
        "conditions": {
            "failed": "last backup failed: cannot finalize workspace content: not implemented",
            "final_backup_complete": 1
        },
        "runtime": {
            "node_name": "foobar"
        }
    }
}